examples/modelscope/README.md

```diff
@@ -0,0 +1,24 @@
+# ModelScope with ITREX
+
+Intel extension for transformers(ITREX) support almost all the LLMs in Pytorch format from ModelScope such as phi,Qwen,ChatGLM,Baichuan,gemma,etc.
+
+## Usage Example
+
+ITREX provides a script that demonstrates the use of modelscope. Run it with the following command:
+```bash
+numactl -m 0 -C 0-55 python run_modelscope_example.py --model_path=qwen/Qwen-7B --prompt=你好
```

Comment:
numactl -l -C xx-xx? if user across sockets? 
Add a note explaining why adding numactl is necessary (to improve performance and teach them how to bind core_id).